src/model/neuroevolution/individual.py

```diff
@@ -101,13 +148,44 @@ def random_neuron_weights_mutation(individual: "ChildIndividual") -> None:
             -3, 3, random_layer.weights.shape[1]
         )
 
+    @staticmethod
+    def neuron_weights_shuffle_mutation(individual: "ChildIndividual") -> None:
+        """
+        Shuffles all weights for a single randomly chosen neuron from a randomly chosen layer.
+
+        Args:
+            individual (ChildIndividual): Individual (neural network) to be modified.
+        """
+        random_layer = individual.get_random_layer()
+        random_neuron_index = np.random.randint(0, random_layer.weights.shape[0])
+        np.random.shuffle(random_layer.weights[random_neuron_index])
+
+    @staticmethod
+    def multiply_shuffle_neuron_weights_mutation(individual: "ChildIndividual") -> None:
+        """
+        Multiplies all weights of one random neuron by random numbers and shuffles them.
+
+        Multiplies all weights for a single randomly chosen neuron,
+        from randomly chosen layer, by random numbers.
+        After weights multiplication by random numbers, the weight are shuffled.
+        There is a random multiplier chosen for every single weight.
```

Comment:
There is a very high level of redundancy of information, only things not described in the summary line have to be expanded upon.

Reply:
Done